climpred/bootstrap.py

```diff
@@ -25,10 +105,18 @@ def _distribution_to_ci(ds, ci_low, ci_high, dim='bootstrap'):
     Returns:
         uninit_hind (xarray object): uninitialize hindcast with hind.coords.
     """
-    # Loads into memory if a dask array.
-    ds = ds.compute()
```

Comment:
this line took much time before, because it always re-computed.

Reply:
Yes we should never compute internally - that should be on the user.